instrumentation/storm/javaagent/src/main/java/io/lumigo/javaagent/instrumentation/storm/StormInstrumentationModule.java

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Is there a way to turn this instrumentation off (env-var)?
we need a way to do it, also document it in the docs

Reply:
We can turn off the instrumentation with this env var `OTEL_INSTRUMENTATION_STORM_ENABLED=false`